docs/wildcam/misc-tools/filter-csv.py

```diff
@@ -0,0 +1,37 @@
+# Filter CSV
+# ----------
+#
+# Reads a CSV file (usually a Zooniverse Subjects export or Classifications
+# export) and filters for rows with specific values.
+#
+# Usage:
+#   python3 filter-csv.py inputFilename.csv outputFilename.csv columnName columnValue
+#
+# Example
+#   "Get all classifications made by user darkeshard"
+#   python3 filter-csv.py gorongosa-classifications.csv output.csv user_name darkeshard
+#
+# (@shaunanoordin 20210322)
+
+import sys, csv
+
+if len(sys.argv) < 5:
+  print('ERROR: not enough arguments')
+  sys.exit()
+
+inputFilename = sys.argv[1]
+outputFilename = sys.argv[2]
+columnName = sys.argv[3]
+columnValue = sys.argv[4]
+
+with open(inputFilename, mode='r', newline='') as inputFile, \
+     open(outputFilename, mode='w+', newline='') as outputFile:
+  
+  inputReader = csv.DictReader(inputFile)
+  outputWriter = csv.DictWriter(outputFile, fieldnames=inputReader.fieldnames)
+  outputWriter.writeheader()
+  
+  for inRow in inputReader:
+    if inRow[columnName] == columnValue:
```

Comment:
you might want to compare the runtimes of this python script vs cmd line tools like `awk`, https://github.com/camallen/cmd-line-data-goodies/blob/master/examples/12_awk.md

```
# use a regex to search for values in column 3
# and print the row num NR and the row $0
awk -F, ' $3 ~ /columnValue/ { print NR, $0 } ' $inputFile

# use a direct string match for values in column 3
awk -F, ' $3 == "columnValue" { print NR, $0 } ' $inputFile
```